cms/src/test/java/uk/nhs/digital/ps/BlankGranularitySelectionFieldValidatorTest.java

```diff
@@ -0,0 +1,159 @@
+package uk.nhs.digital.ps;
+
+import org.apache.wicket.Page;
+import org.apache.wicket.model.IModel;
+import org.apache.wicket.protocol.http.WebApplication;
+import org.apache.wicket.util.tester.WicketTester;
+import org.hippoecm.frontend.model.JcrNodeModel;
+import org.hippoecm.frontend.model.properties.JcrPropertyValueModel;
+import org.hippoecm.frontend.plugin.IPluginContext;
+import org.hippoecm.frontend.plugin.config.IPluginConfig;
+import org.hippoecm.frontend.types.IFieldDescriptor;
+import org.hippoecm.frontend.types.ITypeDescriptor;
+import org.hippoecm.frontend.validation.IFieldValidator;
+import org.hippoecm.frontend.validation.ValidationException;
+import org.hippoecm.frontend.validation.Violation;
+import org.junit.Before;
+import org.junit.Rule;
+import org.junit.Test;
+import org.junit.rules.ExpectedException;
+import org.mockito.Mock;
+
+import javax.jcr.Value;
+import java.util.Collections;
+import java.util.Set;
+import java.util.UUID;
+
+import static java.text.MessageFormat.format;
+import static org.hamcrest.MatcherAssert.assertThat;
+import static org.hamcrest.Matchers.*;
+import static org.hamcrest.core.Is.is;
+import static org.mockito.ArgumentMatchers.eq;
+import static org.mockito.ArgumentMatchers.isA;
+import static org.mockito.BDDMockito.given;
+import static org.mockito.BDDMockito.then;
+import static org.mockito.Mockito.mock;
+import static org.mockito.MockitoAnnotations.initMocks;
+
+public class BlankGranularitySelectionFieldValidatorTest {
+
+    private static final String SUPPORTED_FIELD_TYPE_NAME = "StaticDropdown";
+
+    @Rule
+    public ExpectedException expectedException = ExpectedException.none();
+
+    @Mock
+    private IFieldValidator fieldValidator;
+    @Mock private IFieldDescriptor fieldDescriptor;
+    @Mock private ITypeDescriptor typeDescriptor;
+    @Mock private JcrNodeModel documentNodeModel;
+    @Mock private JcrPropertyValueModel fieldModel;
+    @Mock private IPluginConfig pluginConfig;
+    @Mock private IPluginContext pluginContext;
+    @Mock private IModel<String> violationMessageTranslationModel;
+
+    private BlankGranularitySelectionFieldValidator blankAttachmentFieldValidator;
```

Comment:
Rename variable to "blankGranularitySelectionFieldValidator"?

Reply:
Good spot! I'll rename the field.

Reply:
Done.